app/services/profiles/update.rb

```diff
@@ -23,10 +23,6 @@ def call
       if update_successful?
         @success = true
         @user.touch(:profile_updated_at)
-        # TODO: @citizen428 Preserving a DEV specific feature for now, we should
-        # probably remove this sooner than later as it may not make much sense
-        # for other communities.
-        follow_hiring_tag if SiteConfig.dev_to?
```

Comment:
I dream of the day we can remove `.dev_to?` altogether! We are getting so close 😍

Reply:
Yes @vaidehijoshi, definitely a goal I'm working towards.